pythonforandroid/build.py

```diff
@@ -669,8 +669,12 @@ def run_pymodules_install(ctx, modules):
     venv = sh.Command(ctx.virtualenv)
     with current_directory(join(ctx.build_dir)):
         shprint(venv,
-                '--python=python{}'.format(ctx.python_recipe.major_minor_version_string),
-                'venv')
+                '--python=python{}'.format(
+                    ctx.python_recipe.major_minor_version_string.
+                    partition(".")[0]
```

Comment:
why not loosing the `partition(".")` part to have simply:
```python
'--python=python{}'.format(ctx.python_recipe.major_minor_version_string[0])
```
Also I'm wondering if only the major version is not already accessible in the context somewhere

Reply:
as I explained in chat I think `[0]` is semantically wrong (it happens to work for python with its low versions, but it's just not a correct way to parse a version and basically introduces `>=10` version breakage intentionally for no good reason, IMHO). but that's just my opinion of course, just putting it here for the record :+1: I'm open to changing it to whatever the team agrees on